modules/core/src/main/scala/com/azavea/stac4s/extensions/layer/LayerProperties.scala

```diff
@@ -0,0 +1,22 @@
+package com.azavea.stac4s.extensions.layer
+
+import cats.Eq
+import io.circe.{Decoder, Encoder}
+import io.circe.syntax._
+
+case class LayerProperties(layers: List[String])
+
+object LayerProperties {
+  implicit val eqLayerProperties: Eq[LayerProperties] = Eq.fromUniversalEquals
+
+  implicit val encLayerProperties: Encoder.AsObject[LayerProperties] = Encoder.AsObject.instance[LayerProperties] { o =>
+    Map("layers" -> o.layers.asJson).asJsonObject
+  }
```

Comment:
@jisantuc it looks like property extensions require an AsObject encoder.

Reply:
What do you mean?

Reply:
@jisantuc if properties are a `JsonObject` => there should be a sepecial encoder to support `.asJsonObject` conversion.

Reply:
Ah yeah ok that makes sense